particles/Processing/js/ExpandDims.js

```diff
@@ -8,20 +8,18 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-defineParticle(({DomParticle, log}) => {
+defineParticle(({DomParticle, resolver, log}) => {
+
+  importScripts(resolver(`$here/tf.js`));
 
   return class extends DomParticle {
```

Comment:
Why is it that in other places, we apply the TfMixin, but here we do not?

Reply:
If you want to have access to `set`, I'm pretty sure you need to add the mixin.

Reply:
Yes, thats just a mistake.